scripts/docker/zeppelin-interpreter/Dockerfile

```diff
@@ -30,7 +30,7 @@ ENV VERSION="${version}" \
 RUN set -ex && \
     /usr/bin/apt-get update && \
     DEBIAN_FRONTEND=noninteractive /usr/bin/apt-get install -y openjdk-11-jre-headless wget tini bzip2 && \
-    /usr/bin/wget -qO- https://micromamba.snakepit.net/api/micromamba/linux-64/latest | tar -xvj bin/micromamba && \
+    /usr/bin/wget --no-iri -qO- https://micromamba.snakepit.net/api/micromamba/linux-64/latest | tar -xvj bin/micromamba && \
```

Comment:
How to reproduce the error you are getting?

Reply:
In my case i just run ```docker build -t zeppelin-interpreter:0.12.0-SNAPSHOT -f ./Dockerfile .``` on Docker Engine 25.0.3.
I found the solution [here](https://github.com/mamba-org/mamba/issues/2998#issuecomment-1829292600).

Reply:
Very strange error. I can also reproduce this within Docker.